.github/workflows/api-gateway-integration-test.yml

```diff
@@ -65,57 +65,57 @@ jobs:
 
       - name: "Swagger UI - Request"
         id: "swaggerUI"
-        uses: indiesdev/curl@v1.1
+        uses: Ponnia-M/curl@v1.0.0
```

Comment:
@Ponnia-M should this be within Department of VA GitHub repo rather than your personal repo?

Reply:
I don't have access to fork it into the Department of VA GitHub repo